pkg/objectstore/objectstore_test.go

```diff
@@ -99,8 +100,9 @@ func (s *ObjectStoreProviderSuite) initProvider(c *C, region string) {
 	ctx := context.Background()
 	var err error
 	pc := ProviderConfig{
-		Type:   s.osType,
-		Region: region,
+		Type:     s.osType,
+		Region:   region,
+		Endpoint: s.endpoint,
```

Comment:
We haven't set `s.endpoint`. Is it expected to be `""`?

Reply:
I added this since I used it to test profiles with endpoints. I was planning on following up with adding more buckets to test with, but most other providers have slightly different behavior WRT directories and so not all these tests pass.